src/coreclr/nativeaot/System.Private.CoreLib/src/System/GC.NativeAot.cs

```diff
@@ -70,6 +70,16 @@ public static int GetGeneration(object obj)
             return RuntimeImports.RhGetGeneration(obj);
         }
 
+        public static int GetGenerationSize(int gen)
```

Comment:
Why are these public? It looks like they're just forcing unnecessary entries in CompatibilitySuppressions.xml - these are `internal` in CoreCLR's corelib.